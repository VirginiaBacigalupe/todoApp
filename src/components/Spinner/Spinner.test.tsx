import { shallow } from 'enzyme'
import React from 'react'

import { Spinner } from './index'

describe('Spinner', () => {
  it('Show spinner', () => {
    const wrapper = shallow(<Spinner loading={true} />)
    expect(wrapper).toMatchSnapshot()
  })
  it('Do not show spinner', () => {
    const wrapper = shallow(<Spinner loading={false} />)
    expect(wrapper).toMatchSnapshot()
  })
})
