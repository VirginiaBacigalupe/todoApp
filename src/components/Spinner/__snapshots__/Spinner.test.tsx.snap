// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Do not show spinner 1`] = `
<Modal
  animationType="none"
  hardwareAccelerated={false}
  transparent={true}
  visible={false}
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba( 0 , 0 , 0 , 0.251 )",
        "flex": 1,
        "flexDirection": "column",
        "justifyContent": "space-around",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "rgb(255, 255, 255)",
          "borderRadius": 10,
          "display": "flex",
          "height": 100,
          "justifyContent": "space-around",
          "width": 100,
        }
      }
    >
      <ActivityIndicator
        animating={false}
        color="#999999"
        hidesWhenStopped={true}
        size="small"
      />
    </View>
  </View>
</Modal>
`;

exports[`Spinner Show spinner 1`] = `
<Modal
  animationType="none"
  hardwareAccelerated={false}
  transparent={true}
  visible={true}
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba( 0 , 0 , 0 , 0.251 )",
        "flex": 1,
        "flexDirection": "column",
        "justifyContent": "space-around",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "rgb(255, 255, 255)",
          "borderRadius": 10,
          "display": "flex",
          "height": 100,
          "justifyContent": "space-around",
          "width": 100,
        }
      }
    >
      <ActivityIndicator
        animating={true}
        color="#999999"
        hidesWhenStopped={true}
        size="small"
      />
    </View>
  </View>
</Modal>
`;
